Kinvey/KinveyTests/FileTestCase.swift

```diff
@@ -1077,4 +1077,10 @@ class FileTestCase: StoreTestCase {
         }
     }
     
+    func testGetInstance() {
+        let appKey = "file-get_instance-\(UUID().uuidString)"
+        let client = Client(appKey: appKey, appSecret: "unit-test")
+        let _ = FileStore.getInstance(client)
```

Comment:
basically the crash was happening because the folder for each appKey wasn't not there yet if you call `FileStore.getInstance()` so I'm creating a random appKey to check if the call would not crash